src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -67,9 +67,18 @@ final class PhpStormStubsSourceStubber implements SourceStubber
     /** @var array<string, Node\Stmt\Function_> */
     private $functionNodes = [];
 
-    /** @var array<string, Node\Stmt\Const_|Node\Expr\FuncCall> */
+    /** @var array<string, Node\Stmt\Const_|Node\Expr\FuncCall|null> */
```

Comment:
Could you please add a comment describing `null` here?

Reply:
Done.